### DISCUSSION TITLE
Should we use this for user questions?

### DISCUSSION THREAD

**Discussion by jl-wynen** [scipp/scipp#1467] [General]
@nvaytet 
> This looks like the place to have users post questions, ask for help, see answered questions etc...
Before we made a special Question label in the issues for these kind of things, but should we point users to this space instead?

We can try it out now and see how it works!

**Comment by nvaytet** [comment]
I think this will appear a lot less hacky to users, than having to use a special label in the issues.
Our issues are also full of very technical things that users won't understand nor care about, so this will be much better in my opinion.

**Reply by jl-wynen** [reply]
Agree

**Reply by SimonHeybrock** [reply]
We will probably also have fully technical developer discussions here instead of as issues.

**Comment by jl-wynen** [comment]
I don't see a way to add new categories or tag a discussion. That might make it a bit hard to sort the discussions once there are users with many different questions.

**Reply by nvaytet** [reply]
`Category` on the top right corner?

**Reply by DanNixon** [reply]
There is an edit icon on the main Discussion tab.

**Reply by jl-wynen** [reply]
Sure, but there are only four predefined categories. I was looking for something customisable.

**Reply by nvaytet** [reply]
yes. you can make new categories also
![Screenshot at 2020-12-14 10-27-42](https://user-images.githubusercontent.com/39047984/102064013-10065900-3df7-11eb-8cad-44e8e084d7a2.png)

**Reply by jl-wynen** [reply]
I don't have the pen button.

**Reply by matthew-d-jones** [reply]
Ah, me neither, Neil and Dan must have higher permissions

**Reply by DanNixon** [reply]
Of course, it is one of the golden rules of software development: regardless of the technical ability and competence of the team as a whole, you MUST make an absolute mess of the, in reality, very simple concept of RBAC.

**Reply by nvaytet** [reply]
Dan do you see a button? because you seem to have the same permissions as Jan-Lukas: write, not admin

**Reply by jl-wynen** [reply]
There is a 'Maintainer' tag next to @DanNixon and @nvaytet . But @matthew-d-jones and I are just 'Collaborators'

**Reply by nvaytet** [reply]
weird, i can't see "maintainer", not sure we are looking in the same place.
does that mean you also cannot add new labels in the issues/PRs?

**Reply by jl-wynen** [reply]
I can add labels for issues/PRs.
I am looking at the tags in this discussion. Next to my name it says 'Collaborator' and 'Author'.

**Reply by nvaytet** [reply]
ah you mean on this discussion thread! sorry...

**Reply by nvaytet** [reply]
I think the difference may be because Dan is one of the owners of the Sci++ organisation?

**Comment by matthew-d-jones** [comment]
Looks good to me, just have to remember to make a separate issue ticket if any action arises from the question

**Reply by jl-wynen** [reply]
https://github.com/github/feedback/discussions/2988

**Comment by SimonHeybrock** [comment]
I enabled discussions last week. It is beta, so I don't know if it will stay, i.e., wait before referring users to it.
